src/Preset.php

```diff
@@ -156,7 +157,11 @@ public function run(): void
                 copy(__DIR__ . '/stubs/.eslintrc.js', base_path('.eslintrc.js'));
             });
             $this->command->task('Setup tailwindcss', function () {
-                $this->runCommand('yarn tailwind init');
+                try {
+                    $this->runCommand('yarn tailwind init');
+                } catch (ProcessFailedException $e) {
+                    // Tailwind already setup by the theme
```

Comment:
Suggest we notify the user here that Tailwind already detected.
Are their other circumstances ProcessFailedException would be triggered?

Reply:
I figured it probably wasn't worth writing a line for (as it only happens if you say yes to inertia, which stubs out the tailwind file.

Will update now.